### PR TITLE
Upgrade ktor and other

### PR DESCRIPTION
**Pull request by saibotma**

### CODE REVIEW

File: src/main/kotlin/di/graphQlDi.kt [hunk 1]
```diff
@@ -30,17 +30,14 @@ val graphQlDi = DI.Module("graphql") {
...
-        }
         val config = SchemaGeneratorConfig(
             supportedPackages = listOf(
                 "persistence.jooq.tables.pojos",
                 "clientapi.models",
                 "models",
             ),
             hooks = ChatServerSchemaGeneratorHooks(),
-            dataFetcherFactoryProvider = SimpleKotlinDataFetcherFactoryProvider(objectMapper = objectMapper)
+            dataFetcherFactoryProvider = SimpleKotlinDataFetcherFactoryProvider()
```
Comment (JulianBissekkou) [inline src/main/kotlin/di/graphQlDi.kt:40]: object mapper configuration is not required anymore? 🤔

File: build.gradle.kts [hunk 2]
```diff
@@ -151,18 +162,20 @@ flyway {
     url = postgresUrl
     user = postgresUser
     password = postgresPassword
+    // The default is true, and Flyway gradle plugin will not be executed in production environment.
+    cleanDisabled = false
 }
```
Comment (JulianBissekkou) [inline build.gradle.kts:167]: This update PR looks super painful until now 😆

File: src/main/kotlin/logging/LoggingPlugin.kt [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+import io.ktor.server.request.httpMethod
+import io.ktor.server.request.httpVersion
+import io.ktor.server.request.path
+import io.ktor.server.request.receive
+import io.ktor.util.AttributeKey
+import org.apache.logging.log4j.kotlin.logger
+
+private const val name = "LoggingPlugin"
+val LoggingPlugin = createApplicationPlugin(name = name, createConfiguration = ::LoggingPluginConfig) {
+    val log = logger(name)
```
Comment (JulianBissekkou) [inline src/main/kotlin/logging/LoggingPlugin.kt:16]: new plugin api looks dope 👍🏽

**Review by JulianBissekkou — COMMENTED** [review]
stumbled across this PR, super cool!

File: src/main/kotlin/di/graphQlDi.kt (hunk 1 above)
Comment (JulianBissekkou) [reply src/main/kotlin/di/graphQlDi.kt:40]: Because I saw you still have an object mapper instance and maybe graphql should use the same configuration🤔

Comment (saibotma) [reply src/main/kotlin/di/graphQlDi.kt:40]: It is not possible to pass an object mapper anymore. I still need to test if it works, though. :D

File: build.gradle.kts (hunk 2 above)
Comment (saibotma) [reply build.gradle.kts:167]: Yeah, especially getting Gradle to work (always) takes much time. Dart “pub” is a much better tool, from my point of view.

**Pull request merged**